docs/04-guides/get-viewer-engagement/02-api.mdx

```diff
@@ -0,0 +1,148 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: api
+title: API
+pagination_next: guides/playback-a-video-stream
+---
+
+import Tabs from "./_tabs.mdx";
+import CodeBlock from "@theme/CodeBlock";
+import CodeBlockTabs from "@site/src/components/CodeBlockTabs";
+import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
+
+<Tabs />
+
+The Livepeer API allows you to send video files to Livepeer and get them ready
+for optimized playback. Videos can be provided either by you (static content) or
+your users, given your application offers an interface for them to do so.
+
+## Step 1: Configure your player
```

Comment:
I think this step does not belong on the API tab. You will never see any real viewership data on the dashboard if you don't configure your player to use the viewership plugin (or you use our player).

Maybe this should go before the tabbed display, since it's a common step regardless of whether you're retrieving the metrics from the API or the dashboard.

Maybe separate the doc into 2 sections "register views" (this) and "retrieve views metrics" (which is different for API/dashboard). WDYT?

Reply:
I think i'll move this section out - good point